data/layouts/huion-new-1060-plus.svg

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+   "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg
+   width="360.0"
+   height="240.0"
+   style="color:#000000;font-size:8px;fill:none;stroke:#7f7f7f;stroke-width:0.25"
+   version="1.1"
+   xmlns="http://www.w3.org/2000/svg">
+  <title
+     id="title">Huion New 1060 Plus</title>
+  <path
+     id="ButtonA"
+     class="A Button"
+     d="M 16.0 42.5 H 32.0 v 12.0 H 16.0 c -0.5 0.0 -1.0 -0.5 -1.0 -1.0 v -10.0 c 0.0 -0.5 0.5 -1.0 1.0 -1.0 z" />
+  <path
+     id="LeaderA"
+     class="A Leader"
+     d="M 8.5 51.5 H 13.5" />
+  <text
+     id="LabelA"
+     class="A Label"
+     x="7.0"
+     y="51.5"
+     style="text-anchor:end">A</text>
+  <path
+     id="ButtonB"
+     class="B Button"
+     d="M 49.0 42.5 H 33.0 v 12.0 h 16.0 c 0.5 0.0 1.0 -0.5 1.0 -1.0 v -10.0 c 0.0 -0.5 -0.5 -1.0 -1.0 -1.0 z" />
+  <path
+     id="LeaderB"
+     class="B Leader"
+     d="M 56.5 51.5 H 51.5" />
+  <text
+     id="LabelB"
+     class="B Label"
+     x="57.0"
+     y="51.5"
+     style="text-anchor:start">B</text>
```

Comment:
I think `text-anchor:start` is the right thing for the right buttons but the gnome-settings dialog puts the text to the left of this such that it overlays the button path. Am I wrong here or is this is a bug in gnome?

Reply:
hmm, I think so but I'm not sure myself. firefox and chrome at least appear to render this correctly, so this may be a bug in librsvg. I'd stick with this until proven otherwise ;)